pkg/udev/listentry.go

```diff
@@ -40,8 +40,7 @@ func newUdevListEntry(ptr *C.struct_udev_list_entry) (le *UdevListEntry) {
 	return
 }
 
-// GetNext return UdevListEntry struct if next device present
-// else it returns nil pointer.
+// GetNextEntry returns UdevListEntry struct if next device present else it returns nil pointer.
```

Comment:
Thanks @palash25 for this PR !
Can you please add "is" here "if next device 'is' present" ?